Makefile

```diff
@@ -1,221 +1,254 @@
-##########################################################################################################################
```

Comment:
Not sure why this whole file got changed, I might investigate/remove the commit and just put in changes in a new commit

Reply:
I only added a few lines